Go Client/GameState.cpp

```diff
@@ -63,29 +63,29 @@ stone GameState::WhosMoveJustWent()
 
 uint16_t GameState::PrizonerStones(stone stone)
 {
-	switch (stone)
-	{
-	case stone::white:
+   switch (stone)
+   {
+   case stone::white:
       return m_deadWhiteStones;
       break;
```

Comment:
the break after the return is unreachable code and may cause warnings on some compilers and that sort of thing